deploy/kubernetes/images/prow-gke-release-staging-head/image.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: imagetag-gcepd-driver-prow-head
 imageTag:
   name: gke.gcr.io/gcp-compute-persistent-disk-csi-driver
-  newName: gcr.io/gke-release-staging/gcp-compute-persistent-disk-csi-driver
-  newTag: "latest"
+  newName: gcr.io/k8s-staging-cloud-provider-gcp/gcp-compute-persistent-disk-csi-driver
```

Comment:
Should we also change `name` and its source?

Reply:
That would require changing the integration test code which I'd like to do in its own PR so it doesn't get buried---that sort of thing has had knock-on effects with random test pipelines breaking.

Since we are replacing the image in all overlays probably we should take advantage of doing that to change the placeholder name to something that is clearly a placeholder, which might reduce problems in the future.